ts/features/bonus/cgn/screens/merchants/CgnMerchantsCategoriesSelectionScreen.tsx

```diff
@@ -88,9 +91,22 @@ const CgnTabBar = ({ state, navigation }: MaterialTopTabBarProps) => {
   );
 };
 const CgnMerchantsCategoriesSelectionScreen = () => {
+  const { navigate } =
+    useNavigation<
+      IOStackNavigationProp<CgnDetailsParamsList, "CGN_MERCHANTS_TABS">
+    >();
   useHeaderSecondLevel({
     title: "",
-    supportRequest: true
+    supportRequest: true,
+    secondAction: {
+      icon: "search",
+      onPress() {
+        navigate("CGN_MERCHANTS_SEARCH");
```

Comment:
Could you please use the enumerator `CGN_ROUTES` to navigate to the screen instead of passing a plain string?